src/hotspot/share/utilities/concurrentHashTable.hpp

```diff
@@ -376,18 +387,22 @@ class ConcurrentHashTable : public CHeapObj<F> {
   void delete_in_bucket(Thread* thread, Bucket* bucket, LOOKUP_FUNC& lookup_f);
 
  public:
+  // Default sizes
+  static const size_t DEFAULT_MAX_SIZE_LOG2 = 21;
+  static const size_t DEFAULT_START_SIZE_LOG2 = 13;
+  static const size_t DEFAULT_GROW_HINT = 4; // Chain length
+  static const bool DEFAULT_ENABLE_STATISTICS = true;
```

Comment:
Ok, one last thing. I wonder if we should default to false for enabling table rate statistics?  Only the StringTable and SymbolTable need it.

Reply:
Fixed.